akka-cluster/src/main/resources/reference.conf

```diff
@@ -40,6 +41,15 @@ akka {
     # Disable with "off" or specify a duration to enable.
     down-removal-margin = off
 
+    # Pluggable support for downing of nodes in the cluster.
+    # If this setting is left empty behaviour will depend on 'auto-down-unreachable' in the following ways:
+    # * if it is 'off' the 'NoDowning' provider is used and no automatic downing will be performed
+    # * if it is set to a duration the 'AutoDowning' provider is with the configured downing duration
+    #
+    # If specified the value must be the fully qualified class name of a subclass of
+    # 'akka.cluster.DowningProvider' having a public one argument constructor accepting an 'ActorSystem'
```

Comment:
Actually in the reference.conf it is a mix of ', \` and no quotes at all :/ Will change to \` though.

(And it is `ActorSystem`)